jenkins-xml-to-jobdsl.rb

```diff
@@ -620,17 +620,17 @@ def process(job_name, depth, indent)
       when 'notifyCulprits'
         puts " " * currentDepth + "notifyScmCulprits(#{i.text})"
       when 'notifyOnBuildStart'
-        configureBlock << "'#{i.name}'(#{i.text})"
+        configureBlock << "(ircNode / '#{i.name}').setValue(#{i.text})"
```

Comment:
> Can you make the PR only about the configure block?
I can understand why you would want this, but would rather go over it in detail tomorrow if you have time and then still see if you would like a change to the PR.